packages/e3kit-browser/src/EThree.ts

```diff
@@ -82,7 +87,7 @@ export class EThree extends AbstractEThree {
      */
     async encryptFile(
         file: File | Blob,
-        recipients?: ICard | FindUsersResult | IPublicKey | LookupResult,
+        recipients?: ICard | FindUsersResult | IPublicKey,
         options: EncryptFileOptions = {},
     ): Promise<File | Blob> {
         const chunkSize = options.chunkSize ? options.chunkSize : 64 * 1024;
```

Comment:
'Make sure you pass the resolved value of "EThree.findUsers" or **"EThree.lookupPublicKeys"** methods '